Makefile

```diff
@@ -0,0 +1,4 @@
+CMD_DIR := $(abspath ./cmd)
+
+build-aotutil:
+	cd ${CMD_DIR}/aotutil && make build
```

Comment:
Have you tested this?

Reply:
Yes. Tested it at local and also it would be easier to keep it consistent with [Soaking Github workflow ](https://github.com/aws-observability/aws-otel-collector/blob/main/.github/workflows/soaking.yml#L52-L53)